Source/JavaScriptCore/jit/GPRInfo.h

```diff
@@ -483,8 +483,13 @@ class GPRInfo {
     static constexpr GPRReg wasmScratchGPR1 = X86Registers::r10;
 #endif
     static constexpr GPRReg wasmContextInstancePointer = regCS0;
+#if !OS(WINDOWS)
```

Comment:
Can we swap these cases?

Reply:
It's following the same convention as the rest of the Windows preprocessor directives in this file, should I swap them all?